### ISSUE TITLE
Balance suggestion

### ISSUE THREAD

**Issue by MarwenX3M** [xJon/The-1.12.2-Pack#902]
I think forestry right now is slightly underpowered. Bees hold no ground compared to mystical agriculture, and the energy from peat production needs a boost, as its in most cases inferior to even a passive mekanism generator.

**Comment by xJon** [comment]
I agree, but that doesn't mean it's necessarily a problem. It's more of a gameplay choice thing - play the game you see is right, similarly to how in Tekkit 2 players can opt out of using ProjectE.

**Comment by MarwenX3M** [comment]
Okay, thats fair. Its just such a discrepancy, making peat is almost like comical. But I guess that has been for a long while, ever since RF was introduced. The arboretum on the other hand is quite powerful with how much wood it can produce I think. Especially early game with how little power forestry appliances use. Not unbalanced or anything, but definitely viable.
